english/forest/scream/run/run.md

```diff
@@ -0,0 +1 @@
+Since you know you cannot outrun a tiger. You decide to run toward it making as much noise as possible in a way to scare it off!!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.